architecture/adr-010-built-in-apis.md

```diff
@@ -0,0 +1,39 @@
+# Architecture Decision Record: Built-in APIs should be built on top of the Message Router
+
+## Context
+
+ComposeUI as a framework will provide a set of standard APIs to modules. 
+One of these standard APIs is the pub/sub/invoke functionality of the Message Router.
+Modules will communicate and provide services for each other via the Message Router.
+However, there might be other framework services that we provide, such as login,
```

Comment:
"there might be other framework service APIs that Compose provides"

Suggest adding API after service for consistency.  Also, suggest "Compose" instead of "we", I feel it's more clear